project/en-US/stm/stm.xml

```diff
@@ -144,8 +210,13 @@ else
             <para>Some of the discussions around AtomicAction etc. are deliberately brief here because you can find more
                 information in the relevant ArjunaCore documentation.</para>
         </note>
-        <para>First we need to create an STM Container: this is the entity which represents the transactional memory
-            within which each object will be maintained. We need to tell each Container about the type of objects for
+        <para>In order for the STM subsytem to have knowledge about which classes are to be managed within the context of
+            transactional memory it is necessary to provide a minimal level of instrumentation. This occurs by categorising
+            STM-aware and STM-unware classes through an interface bounday; specifically all STM-aware objects must be
```

Comment:
STM-unware/STM-unaware

Reply:
bounday/boundary

Reply:
Not sure what you mean by "STM-unware/STM-unaware"

Reply:
I meant, would it not be "STM-unaware"? Maybe I am wrong but something "-unware" doesn't seem right to me?

Reply:
No it's definitely correct. There are STM-aware classes and since not every class can be used in STM there must therefore be STM-unaware classes too.